src/app/types.ts

```diff
@@ -0,0 +1,120 @@
+export interface ItemCardProps {
```

Comment:
When you need to define component specific interfaces (like props), add the interface directly in the file where component is